locales/si/validation-inline.php

```diff
@@ -40,23 +40,23 @@
     'digits'               => 'මෙම විය යුතුය :digits ඉලක්කම්.',
     'digits_between'       => 'මේ අතර විය යුතුයි :min හා :max ඉලක්කම්.',
     'dimensions'           => 'මෙම රූපය සාවද්ය මාන ඇත.',
-    'distinct'             => 'මේ ක්ෂේත්රයේ ඇති duplicate වටිනාකම.',
+    'distinct'             => 'මේ ක්ෂේත්‍රය ඇති duplicate වටිනාකම.',
     'email'                => 'මෙම විය යුතුය, වලංගු විද්යුත්-තැපැල් ලිපිනය.',
     'ends_with'            => 'මේ අවසන් විය යුතුය සමග එක් පහත සඳහන්: :values.',
     'exists'               => 'තෝරාගත් අගය සාවද්ය වේ.',
     'file'                 => 'අන්තර්ගත විය යුතුය ගොනුව.',
-    'filled'               => 'මේ ක්ෂේත්රයේ ඇති කළ යුතු අගය.',
-    'gt'                   => [
```

Comment:
don't delete space please

Reply:
Sorry about that, probably due to automatic formatting. I'll fix it.